src/osdc/Objecter.h

```diff
@@ -1105,6 +1105,21 @@ struct ObjectOperation {
     encode(src, osd_op.indata);
     encode(src_oloc, osd_op.indata);
   }
+  void copy_from2(object_t src, snapid_t snapid, object_locator_t src_oloc,
+		 version_t src_version, unsigned flags,
+		 uint32_t truncate_seq, uint64_t truncate_size,
+		 unsigned src_fadvise_flags) {
+    using ceph::encode;
+    OSDOp& osd_op = add_op(CEPH_OSD_OP_COPY_FROM);
```

Comment:
CEPH_OSD_OP_COPY_FROM2?

Reply:
Oh, right.  Thanks for catching this.  I've just pushed a fixed version into my branch.